adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/FeatureRDD.scala

```diff
@@ -327,8 +327,23 @@ case class FeatureRDD(rdd: RDD[Feature],
    * @return Since a feature maps directly to a single genomic region, this
    *   method will always return a Seq of exactly one ReferenceRegion.
    */
-  protected def getReferenceRegions(elem: Feature): Seq[ReferenceRegion] = {
-    Seq(ReferenceRegion.unstranded(elem))
+  protected def getReferenceRegions(elem: Feature,
+                                    stranded: Boolean = false): Seq[ReferenceRegion] = {
+    stranded match {
+      case true => {
+        Seq({
+          try {
+            ReferenceRegion.stranded(elem)
+          } catch {
+            case e: IllegalArgumentException => ReferenceRegion.unstranded(elem)
```

Comment:
This requirement was recently [added](https://github.com/bigdatagenomics/adam/commit/e98ee2d1c5582364673fada70779aab5a73856b1#diff-521713a7bcc29cc21ce94e2f9cd722bfR294), and I'm not sure that we should remove it. I don't mind getting rid of the `try..catch` block, but I think that a `null` field should at least be logged. Maybe `ValidationStringency` here?